language-plutus-core/language-plutus-core.cabal

```diff
@@ -202,9 +235,11 @@ executable language-plutus-core-generate-evaluation-test
         text -any
 
 test-suite language-plutus-core-test
+    import: lang
```

Comment:
What's that?

Reply:
It's a cabal common stanza, which I added at the top. You can then import (include) it in other components. Avoids repeating the set of default extensions over and over. `lang` isn't a great name, but it's what we use in other packages (I think because it originally just had the `default-language` in it)